conductor_api/src/interface.rs

```diff
@@ -135,8 +135,10 @@ impl ConductorApiBuilder {
             let hc_lock = instance.clone();
             let hc_lock_inner = hc_lock.clone();
             let mut hc = hc_lock_inner.write().unwrap();
-            let call_params = params_map.get("params");
-            let params_string = serde_json::to_string(&call_params)
+
+            // Getting thee arguments of the call contained in the json-rpc 'params'
+            let call_args = params_map.get("args");
```

Comment:
Can we make it so that it falls back to still checking "params", and also perhaps prints a deprecation warning log message? Otherwise this will be a significant breaking change.

You can use `hc.context().log()` for the warning.

Reply:
Sure! Ok thanks for the advice! I just pushed a fallback handling to the old `params` implementation and add deprecation warning